test/jdk/sun/text/resources/LocaleDataTest.java

```diff
@@ -330,6 +330,9 @@ static boolean processLine(String key, String expectedValue, PrintWriter out,
                     || rbName.equals("LocaleNames")
                     || rbName.equals("TimeZoneNames")) {
                 fullName = UTIL_RESOURCES_PACKAGE + "." + rbName;
+            } else if (rbName.equals("CollationData")) {
```

Comment:
CLDR does not define CollationData, so this type of check actually needs to be removed, since we should not be including any CLDR data in the LocaleData.cldr file in the first place.